perceval/backends/core/git.py

```diff
@@ -245,41 +254,47 @@ def __fetch_from_log(self):
                     self.uri, self.gitpath)
         return self.parse_git_log_from_file(self.gitpath)
 
-    def __fetch_from_repo(self, from_date, branches, latest_items=False):
+    def __fetch_from_repo(self, from_date, to_date, branches, latest_items=False):
         # When no latest items are set or the repository has not
         # been cloned use the default mode
         default_mode = not latest_items or not os.path.exists(self.gitpath)
 
         repo = self.__create_git_repository()
 
         if default_mode:
-            commits = self.__fetch_commits_from_repo(repo, from_date, branches)
+            commits = self.__fetch_commits_from_repo(repo, from_date, to_date, branches)
         else:
             commits = self.__fetch_newest_commits_from_repo(repo)
 
         return commits
 
-    def __fetch_commits_from_repo(self, repo, from_date, branches):
+    def __fetch_commits_from_repo(self, repo, from_date, to_date, branches):
         if branches is None:
             branches_text = "all"
         elif len(branches) == 0:
             branches_text = "no"
         else:
             branches_text = ", ".join(branches)
 
-        logger.info("Fetching commits: '%s' git repository from %s; %s branches",
-                    self.uri, str(from_date), branches_text)
-
         # Ignore default datetime to avoid problems with git
         # or convert to UTC
+        if to_date == DEFAULT_LAST_DATETIME:
+            logger.info("Fetching commits: '%s' git repository from %s to CURRENT DATETIME; %s branches",
```

Comment:
I suggest to move this logging message outside the `if` block. It makes more sense there and we'll only need one message. Chance `CURRENT DATETIME` by the value of `to_date`. I'd rather be clear on what parameters the backend is using to fetch new items.